package.json

```diff
@@ -92,6 +94,7 @@
     "mkdirp": "^0.5.1",
     "npm-run-all": "^4.1.3",
     "prettier": "^2.0.5",
+    "raw-loader": "~4.0.0",
```

Comment:
If I don't include raw-loader, I get the following compilation error (which makes me think raw-loader is getting used instead of file-loader, which is what I want)

```
Error: resolve 'raw-loader' in '/Users/alli.scibisz/nbsv'
  Parsed request is a module
  using description file: /Users/alli.scibisz/nbsv/package.json (relative path: .)
    resolve as module
      looking for modules in /Users/alli.scibisz/nbsv/node_modules
        single file module
          using description file: /Users/alli.scibisz/nbsv/package.json (relative path: ./node_modules/raw-loader)
            no extension
              /Users/alli.scibisz/nbsv/node_modules/raw-loader doesn't exist
            .js
              /Users/alli.scibisz/nbsv/node_modules/raw-loader.js doesn't exist
        /Users/alli.scibisz/nbsv/node_modules/raw-loader doesn't exist
      /Users/alli.scibisz/node_modules doesn't exist or is not a directory
      /Users/node_modules doesn't exist or is not a directory
      /node_modules doesn't exist or is not a directory
    at compilerCallback (/Users/alli.scibisz/nbsv/node_modules/@jupyterlab/builder/lib/build-labextension.js:100:19)
    at /Users/alli.scibisz/nbsv/node_modules/@jupyterlab/builder/lib/build-labextension.js:142:21
    at /Users/alli.scibisz/nbsv/node_modules/neo-async/async.js:2830:7
    at done (/Users/alli.scibisz/nbsv/node_modules/neo-async/async.js:2865:11)
    at /Users/alli.scibisz/nbsv/node_modules/neo-async/async.js:2818:7
    at /Users/alli.scibisz/nbsv/node_modules/@jupyterlab/builder/node_modules/webpack/lib/HookWebpackError.js:68:3
    at Hook.eval [as callAsync] (eval at create (/Users/alli.scibisz/nbsv/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:15:1)
    at Hook.CALL_ASYNC_DELEGATE [as _callAsync] (/Users/alli.scibisz/nbsv/node_modules/tapable/lib/Hook.js:18:14)
    at Cache.shutdown (/Users/alli.scibisz/nbsv/node_modules/@jupyterlab/builder/node_modules/webpack/lib/Cache.js:150:23)
    at /Users/alli.scibisz/nbsv/node_modules/@jupyterlab/builder/node_modules/webpack/lib/Compiler.js:1229:15
```